tests/test_normalizers.py

```diff
@@ -33,25 +34,33 @@ def test_deeparg_normalizer(hamronized):
 
     pd.testing.assert_frame_equal(normed, golden_file)
 
-@pytest.mark.parametrize('database', ['argannot', 'megares', 'ncbi', 'resfinder'])
+@pytest.mark.parametrize('database', ['argannot', 'megares', 'ncbi', 'resfinder', 'random_db'])
 def test_abricate_normalizer_hamronized(database):
-    normalizer = argnorm.AbricateNormalizer(database=database, is_hamronized=True)
-    input_path = f'./examples/hamronized/abricate.{database}.tsv'
+    try:
+        normalizer = argnorm.AbricateNormalizer(database=database, is_hamronized=True)
+    except:
+        warn(f'Unsupported database: {database} used.')
+    else:
+        input_path = f'./examples/hamronized/abricate.{database}.tsv'
 
-    normed = get_normed(normalizer, input_path)
-    golden_file = pd.read_csv(os.path.join('./outputs/', 'hamronized', f'abricate.{database}.tsv'), sep='\t')
+        normed = get_normed(normalizer, input_path)
+        golden_file = pd.read_csv(os.path.join('./outputs/', 'hamronized', f'abricate.{database}.tsv'), sep='\t')
 
-    pd.testing.assert_frame_equal(normed, golden_file)
+        pd.testing.assert_frame_equal(normed, golden_file)
 
-@pytest.mark.parametrize('database', ['argannot', 'megares', 'ncbi'])
+@pytest.mark.parametrize('database', ['argannot', 'megares', 'ncbi', 'sarg', 'random_db'])
 def test_abricate_normalizer_raw(database):
-    normalizer = argnorm.AbricateNormalizer(database=database, is_hamronized=False)
-    input_path = f'./examples/raw/abricate.{database}.tsv'
+    try:
```

Comment:
Won't this hide errors in the testing?

You should be testing that `random_db` triggers an error (by explicitly failing if it doesn't)